express/routes/index.js

```diff
@@ -0,0 +1,35 @@
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const express = require('express');
+const fs = require('fs');
+const path = require('path');
+
+const errorHandler = require('../middleware/error-handler');
```

Comment:
Looks like a lot of things are mixed here, lets separate them to different files